### PR TITLE
mount: Added Retract Mount2

### PR DESCRIPTION
**Pull request by muramura**
https://github.com/ArduPilot/ardupilot/pull/27352

AFTER
Screenshot from 2024-07-03 21-09-38

### CODE REVIEW

**Comment by Hwurzburg** [comment]
tnx...this page also needs to be updated: https://ardupilot.org/copter/docs/common-auxiliary-functions.html

**Comment by muramura** [comment]
@Hwurzburg san.  added pages.

AFTER
Screenshot from 2024-07-05 00-50-54

Screenshot from 2024-07-05 00-51-15

Screenshot from 2024-07-05 00-53-09

Screenshot from 2024-07-05 00-59-36

Screenshot from 2024-07-05 01-01-48

Screenshot from 2024-07-05 01-02-45

File: copter/source/docs/channel-7-and-8-options.rst [hunk 1]
```diff
@@ -233,7 +233,17 @@ Reset :ref:`Auto <auto-mode>` to run the first mission command in the command li
    </td>
    </tr>
    <tr>
-   <td><strong>Retract Mount</strong></td>
+   <td><strong>Retract Mount1</strong></td>
```
Comment (Hwurzburg) [inline copter/source/docs/channel-7-and-8-options.rst:236]: no need to make these changes...its a deprecated page and they are not updated

**Review by Hwurzburg — CHANGES_REQUESTED** [review]
thanks for adding the updates....one should not be done since its on a deprecated page that is not longer valid

File: copter/source/docs/channel-7-and-8-options.rst (hunk 1 above)
Comment (muramura) [reply copter/source/docs/channel-7-and-8-options.rst]: @Hwurzburg san.  I understand. I have reverted the change.

File: common/source/docs/common-auxiliary-functions.rst [hunk 2]
```diff
@@ -374,7 +376,8 @@ Other functions are:
                                          | For developers only.
     AttCon Accel Limits                  | Turns on/off attitude controller acceleration limits.
                                          | For developers only.
-    Retract Mount                         Move the :ref:`camera mount <common-cameras-and-gimbals>` to its retracted position.
+    Retract Mount1                        Move the :ref:`camera mount <common-cameras-and-gimbals>` to its retracted position.
+    Retract Mount2                        Move the :ref:`camera mount <common-cameras-and-gimbals>` to its retracted position.
```
Comment (Hwurzburg) [inline common/source/docs/common-auxiliary-functions.rst:380]: move retract mnt2mnt2 to between SwitchExternalAHRS     and   ARM/DISARM (4.2 and higher)

**Review by Hwurzburg — CHANGES_REQUESTED** [review]

**Pull request merged**